src/pkg/cli/compose/baseimage.go

```diff
@@ -22,6 +23,10 @@ func FindAllBaseImages(project *composeTypes.Project) ([]string, error) {
 			dockerfileFullPath := service.Build.Context + string(os.PathSeparator) + dockerfilePath
 			images, err := extractDockerfileBaseImages(dockerfileFullPath)
 			if err != nil {
+				if os.IsNotExist(err) {
+					term.Debugf("dockerfile %s does not exist for service %s", dockerfileFullPath, service.Name)
```

Comment:
We use this format for all warn/debug messages:
```suggestion
					term.Debugf("service %q: dockerfile %q does not exist; skipping", service.Name, dockerfileFullPath)
```